app/views/stories/_listdetail.html.erb

```diff
@@ -7,7 +7,7 @@ class="story <%= story.vote && story.vote[:vote] == 1 ? "upvoted" : "" %>
 <%= story.is_hidden_by_cur_user ? "hidden" : "" %>
 <%= story.is_saved_by_cur_user ? "saved" : "" %>
 <%= story.is_expired? ? "expired" : "" %>">
-<div class="story_liner">
+<div class="story_liner h-entry">
```

Comment:
I'm sure in a year or two I'm going to forget what these extra classes are for, notice they have no rules in the css, and remove them. Could you add a comment with all four classes (so it shows up in any grep) and linking to the webmention docs?